Vagrantfile

```diff
@@ -185,14 +185,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.proxy.enabled = { docker: false }
   end
 
+  etcd_seed_cluster = Net::HTTP.get(URI.parse("https://discovery.etcd.io/new?size=%d" % (NODES.to_i + 1)))
```

Comment:
Not really. If the worker nodes are only proxies, there's no need to define multiple etcd instances. This is a development tool that doesn't care about etcd high-availability (redundancy in this case is a waste of resources).